.release/provider-schema.json

```diff
@@ -0,0 +1,7647 @@
+            "description": "Object that represents datastore of backup",
+            "computed": true,
+            "elem": {
+              "schema": {
+                "type": {
+                  "type": "TypeString",
+                  "description": "Version of the datastore. Changing this creates a new instance.",
+                  "computed": true
+                },
+                "version": {
+                  "type": "TypeString",
+                  "description": "Type of the datastore. Changing this creates a new instance.",
+                  "computed": true
+                }
+              }
+            }
+          },
+          "dbms_id": {
+            "type": "TypeString",
+            "required": true,
+            "description": "ID of the instance or cluster, to create backup of.",
+            "force_new": true
+          },
+          "dbms_type": {
+            "type": "TypeString",
+            "description": "Type of dbms for the backup, can be \"instance\" or \"cluster\".",
+            "computed": true
+          },
+          "description": {
+            "type": "TypeString",
+            "optional": true,
+            "description": "The description of the backup",
+            "force_new": true
+          },
+          "location_ref": {
+            "type": "TypeString",
+            "description": "Location of backup data on backup storage",
+            "computed": true
+          },
+          "meta": {
+            "type": "TypeString",
+            "description": "Metadata of the backup",
+            "computed": true
+          },
+          "name": {
+            "type": "TypeString",
+            "required": true,
+            "description": "The name of the backup. Changing this creates a new backup",
+            "force_new": true
+          },
+          "size": {
+            "type": "TypeFloat",
+            "description": "Backup's volume size",
+            "computed": true
+          },
+          "updated": {
+            "type": "TypeString",
+            "description": "Timestamp of backup's last update",
+            "computed": true
+          },
+          "wal_size": {
+            "type": "TypeFloat",
+            "description": "Backup's WAL volume size",
+            "computed": true
+          }
+        },
+        "timeouts": {
+          "create": 30,
+          "delete": 30
+        }
+      },
+      "vkcs_db_cluster": {
+        "schema": {
+          "availability_zone": {
+            "type": "TypeString",
+            "optional": true,
+            "description": "The name of the availability zone of the cluster. Changing this creates a new cluster.",
+            "force_new": true
+          },
+          "backup_schedule": {
+            "type": "TypeList",
+            "optional": true,
+            "description": "Object that represents configuration of PITR backup. This functionality is available only for postgres datastore. **New since v.0.1.4**",
```

Comment:
don't we want descriptions to be cleaned from "new since" in the scheme?

Reply:
Removed.